homework-submission/week4-class16/src/stores/TodosStore.jsx

```diff
@@ -0,0 +1,90 @@
+import { observable, computed, action } from "mobx";
+import Data from "../Data.json";
+import { configure } from "mobx";
+configure({ enforceActions: true });
+
+class TodosStore {
+  @observable
+  edit = false;
+  @observable
+  Data = Data;
+
+  @action
+  handleCheck = todoId => {
+    const newData = this.Data.map(todo => {
+      if (todo.id === todoId) {
+        return { ...todo, done: !todo.done };
```

Comment:
Great use of the spread operator, but you can leave it in MobX, in state it's very important to not mutate it and that's why I was using it always and others around.

if you chosed to use it here you may use it everywhere or no where, in order to not confuse yourself where you have to use it, or others they have to read es6 or es5